ar.xjuan.Cambalache.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id" : "ar.xjuan.Cambalache",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "separate-locales": false,
+    "command" : "cambalache",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
```

Comment:
does it really need network access?

Reply:
sort of, only localhost to access broadway backend